### DISCUSSION TITLE
Overexposed saturation

### DISCUSSION THREAD

**Discussion by el-bakkali** [awawa-dev/HyperHDR#957] [Q&A] [answered]
Hi
i recently purchase the ecozee splitter with MS2130 capture card
i flashed the ms2130 with experimental firmware hyperhdr f3
i confirmed by using OBS i get 120hz and 24hz modes

Anyway i plugged it into my orange pi 3b and configured
my setup is as follows
Firestick 4k(hdmi in)
LG tv 4k(hmdi out1) scaler switch disabed
MS2130(hdmi out2) scaler switch enabled
Edid copy switch

the picture without tone mapping is fine
However when i enable tone mapping and test via firestick i see following:

The mans face is red and seems over saturated in colour
![image](https://github.com/user-attachments/assets/92988693-ee8e-47f8-b105-d2ff58701e35)

also is it normal for firestick 4k diagnostics to not show output of hdr10
when i plug the firestick directly to my tv is is recongised, however the ecozee splitters seems not to pass this through
<img width="700" alt="Screenshot 2024-10-15 at 16 00 51" src="https://github.com/user-attachments/assets/6237a3c1-7c9d-4874-99a1-388886481245">

**Comment by awawa-dev** [comment]
Hi

>the picture without tone mapping is fine

If you use default (or dedicated for ms2130) LUT that strongly indicates that the signal is SDR

**Reply by el-bakkali** [reply]
I downloaded the LUT ms2130

are you saying if I have downloaeded that and applied it. I don’t need to turn on tone mapping ?

**Comment by awawa-dev** [comment]
Which model of Ezcoo do you have? Did you try to force HDR mode using Ezcoo switches instead of EDID copy?

**Reply by el-bakkali** [reply]
I have the sp12h2

yes I used the switch as per above

HDMI1 out to lg switch is off
Hmdi2 out to grabber is enabled

**Reply by awawa-dev** [reply]
I meant using DIP switches to set constant EDID that supports HDR.

**Reply by el-bakkali** [reply]
Yes I’m using the white dip switches 
I wasn’t aware of any other way

HDMI1 output dip switch is off
HDMI2 out to grabber is on

**Reply by awawa-dev** [reply]
This one?
![obraz](https://github.com/user-attachments/assets/9b8b1249-2ac5-4101-b547-28df3a319eaf)

**Reply by el-bakkali** [reply]
This is how i have mine set

![image](https://github.com/user-attachments/assets/ebe48467-ee17-4d6a-9592-3ba8e2308bf5)
![image](https://github.com/user-attachments/assets/2af9d33d-db00-4987-b335-24770cba4d72)

**Reply by awawa-dev** [reply]
set it to 4K7.1 or 4K5.1 or test both

**Reply by el-bakkali** [reply]
perfect, thank you so much
its working now
i set it to 4k5.1

I hope this helps other people in future too
i was following your M2130 discussion
and had it set different by i think you only need copy if you are using dolby vision right ?

**Reply by awawa-dev** [reply]
I used COPY for my Firestick and LG but I was testing DV not HDR10.. at least I dont remember for sure.

This is rather strange behavior and I can see several possible causes:

1 the splitter did not refresh the EDID after connecting the TV to OUTPUT1 while COPY was enabled...maybe it needed a hard reset?

2 Ezcoo can modify the EDID if COPY is enabled...maybe something went wrong with its firmware algorithm? You can try without the grabber connected (only TV to OUTPUT 1) after a reset with the setting set to COPY.

3 the splitter is damaged and the COPY EDID settings are frozen.

**Reply by el-bakkali** [reply]
interesting i think first option theory one,

while connected, I removed grabber hmdi cable
switched to copy and it remained working passing through hdr10

I then reconnected the grabber hdmi cable and it remained working
![image](https://github.com/user-attachments/assets/81e2fd70-e463-4e4b-9590-f0c435707974)

this is with copy switched on now

**Reply by awawa-dev** [reply, marked as answer]
You must reset some device for example splitter to force EDID handshake again. Otherwise they will stick to current settings.

**Reply by el-bakkali** [reply]
this is good to know for future, thank you for educating me on this
Thank you for your support and efforts on this project
its made my viewing experience truly amazing

**Reply by awawa-dev** [reply]
Thank you! ❤️
